validator-same-as-directive.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.directive('brValidatorSameAs', factory);
+}
+
+/* @ngInject */
+function factory() {
+  return {
+    restrict: 'A',
+    require: 'ngModel',
+    scope: {
+      sameAs: '<brValidatorSameAs'
+    },
+    link: Link
+  };
+
+  function Link(scope, element, attrs, ctrl) {
+    scope.$watch('sameAs', function() {
+      ctrl.$validate();
+    });
+
+    ctrl.$validators.brValidatorSameAs = function(modelValue) {
+      if(scope.sameAs !== modelValue) {
+        return false;
+      }
+    };
```

Comment:
This will return undefined or false.  The docs say return true or false.  How about just:
```js
    ... = function(modelValue) {
      return scope.sameAs === modelValue;
    }
```
Looking at docs https://docs.angularjs.org/guide/forms, do you need the $isEmpty handling?  Or the viewValue param check?